README.md

```diff
@@ -1,2 +1,3 @@
 # git-demo
 to demonstrate how to use git and GitHub
+Georgia Gans github.com/georgiagans
```

Comment:
Please add a `-` before your name, so that this becomes a list item.

Reply:
Also, please make your name into a link that goes to that URL. The URL should not be visible.